modules/virt-must-gather-options.adoc

```diff
@@ -33,75 +33,81 @@ Using too many parallel processes can cause performance issues. Increasing the m
+`/usr/bin/gather`:: Use the default `must-gather` script, which collects cluster data from all namespaces and includes only basic VM information. This script is compatible only with the `PROS` variable.
 
-`gather`:: Use the default `must-gather` script, which collects cluster data from all namespaces and includes only basic VM information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --vms_details`:: Collect VM log files, VM definitions, control-plane logs, and namespaces (and their child objects) that belong to {VirtProductName} resources. If you use this parameter without specifying a namespace or VM, the `must-gather` tool collects this data for all VMs in the cluster. This script is compatible with all environment variables, but you must specify a namespace if you use the `VM` variable.
 
-`gather_images`:: Collect image and image stream custom resource information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --images`:: Collect image, image-stream-tags, and image-stream custom resource information. This script is compatible only with the `PROS` variable.
 
 [id="usage-and-examples_{context}"]
 == Usage and examples
 
 Environment variables are optional. You can run a script by itself or with one or more compatible environment variables.
 
+To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
+
 [cols="1a,1a", options="header"]
 .Compatible parameters
 |===
 |Script |Compatible environment variable
-|`gather_vms_details`
+|`/usr/bin/gather`   |* `PROS=<number_of_processes>`
+|`/usr/bin/gather --vms_details`
 |* For a namespace: `NS=<namespace_name>`
 
 * For a VM: `VM=<vm_name> NS=<namespace_name>`
 
 * `PROS=<number_of_processes>`
 
-|`gather`   |* `PROS=<number_of_processes>`
-|`gather_images`   |* `PROS=<number_of_processes>`
+
+|`/usr/bin/gather --images`   |* `PROS=<number_of_processes>`
 |===
 
-To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
+
 
 .Syntax
 
+
+
 [source,terminal,subs="attributes+"]
 ----
 $ oc adm must-gather \
   --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel9:v{HCOVersion} \
   -- <environment_variable_1> <environment_variable_2> <script_name>
 ----
 
-.Detailed VM information
+.Default data collection parallel processes
 
-The following command collects detailed VM information for the `my-vm` VM in the `mynamespace` namespace:
+By default, five processes run in parallel. You can change this default number; for example, the following command collects default `must-gather` information by using a maximum of three parallel processes:
```

Comment:
I recommend removing this sentence and adding the default as a callout:

----
  -- PROS =3 /usr/bin/gather <1>
----
<1> The default number of process is 5.

Reply:
Leaving as is.

Reply:
Since this is a change that you made, I think it is reasonable to request that you change it.

Reply:
A bit more feedback on this. The label states that this is the default. However, the example specifies `3`, which is not the default. I suggest you document the default in the sample and use your new sentence as the callout.